aptos-move/framework/move-stdlib/sources/bit_vector.move

```diff
@@ -198,42 +218,55 @@ module std::bit_vector {
-                };
-                i < bitvector.length
-            }) {
+                    spec {
+                        invariant i >= amount;
+                        invariant bitvector.length == old(bitvector).length;
+                        invariant forall j in amount..i: old(bitvector).bit_field[j] == bitvector
+                            .bit_field[j - amount];
+                        invariant forall j in (i - amount)..bitvector.length: old(
+                            bitvector
+                        ).bit_field[j] == bitvector.bit_field[j];
+                        invariant forall k in 0..i - amount: bitvector.bit_field[k] == old(
+                            bitvector
+                        ).bit_field[k + amount];
+                    };
+                    i < bitvector.length
+                }) {
                 if (is_index_set(bitvector, i)) set(bitvector, i - amount)
                 else unset(bitvector, i - amount);
                 i = i + 1;
             };
 
-
             i = bitvector.length - amount;
 
             while ({
-                spec {
-                    invariant forall j in bitvector.length - amount..i: !bitvector.bit_field[j];
-                    invariant forall k in 0..bitvector.length - amount: bitvector.bit_field[k] == old(bitvector).bit_field[k + amount];
-                    invariant i >= bitvector.length - amount;
-                };
-                i < bitvector.length
-            }) {
+                    spec {
+                        invariant forall j in bitvector.length - amount..i: !bitvector.bit_field[j];
+                        invariant forall k in 0..bitvector.length - amount: bitvector.bit_field[k] ==
+                             old(bitvector).bit_field[k + amount];
+                        invariant i >= bitvector.length - amount;
+                    };
+                    i < bitvector.length
+                }) {
                 unset(bitvector, i);
                 i = i + 1;
             }
         }
     }
+
     spec shift_left_for_verification_only {
         aborts_if false;
-        ensures amount >= bitvector.length ==> (forall k in 0..bitvector.length: !bitvector.bit_field[k]);
+        ensures amount >= bitvector.length ==>
+            (forall k in 0..bitvector.length: !bitvector.bit_field[k]);
         ensures amount < bitvector.length ==>
-            (forall i in bitvector.length - amount..bitvector.length: !bitvector.bit_field[i]);
+            (
+                forall i in bitvector.length - amount..bitvector.length: !bitvector.bit_field[i]
+            );
         ensures amount < bitvector.length ==>
-            (forall i in 0..bitvector.length - amount: bitvector.bit_field[i] == old(bitvector).bit_field[i + amount]);
+            (
+                forall i in 0..bitvector.length - amount: bitvector.bit_field[i] == old(
```

Comment:
It is not possible to avoid line breaks in old(...), but for the forall statement, there are alternative ways to make the formatting more aesthetically pleasing, such as breaking the line after 'amount:', which we have already considered. Due to time constraints, it has not yet been implemented in the short term.